### ISSUE TITLE
Won't install on ubuntu for windows

### ISSUE THREAD

**Issue by BEllis** [josephg/node-foundationdb#25]
I get the following make error, I'm assuming I need to install a library or apk package but I can't find anything suitable. I installed foundationDB for windows, but I guess the libraries are .dlls for the windows version?

make: Entering directory '/mnt/c/dev/.../node_modules/foundationdb/build'
  CXX(target) Release/obj.target/fdblib/src/FdbV8Wrapper.o
In file included from ../src/Database.h:28:0,
                 from ../src/FdbV8Wrapper.cpp:31:
../src/Transaction.h:29:32: fatal error: foundationdb/fdb_c.h: No such file or directory
compilation terminated.
fdblib.target.mk:109: recipe for target 'Release/obj.target/fdblib/src/FdbV8Wrapper.o' failed
make: *** [Release/obj.target/fdblib/src/FdbV8Wrapper.o] Error 1
make: Leaving directory '/mnt/c/dev/.../node_modules/foundationdb/build'

**Comment by josephg** [comment]
Hm. You’ll need to install the Foundationdb client library dpkg inside Linux for windows. But as for the error, I’m surprised - it should just be using the normal Linux prebuilds. It shouldn’t need to compile at all.

**Comment by josephg** [comment]
I just tried this on ubuntu 18.04 on windows. I ran into a similar error, but it went away after I installed the foundationdb-clients dpkg file from [the foundationdb download page](https://www.foundationdb.org/download/).

It looks like this package is trying to load the linux foundationdb client library. When that fails, it assumes it hasn't been compiled for your platform and then it tries to re-compile everything. And the recompilation fails because you're missing the headers - since again, you don't have the client installed.

So running this made it work just fine for me:

```
$ wget 'https://www.foundationdb.org/downloads/6.0.18/ubuntu/installers/foundationdb-clients_6.0.18-1_amd64.deb'
$ sudo dpkg -i foundationdb-clients_6.0.18-1_amd64.deb
$ npm i foundationdb
```

Let me know if you still have problems. Otherwise close this issue
